packages/protocol/scripts/check-backward.ts

```diff
@@ -81,6 +88,17 @@ try {
     out
   )
 
+  try {
+    const version = getReleaseVersion(argv.new_branch)
+    if (version === 11) {
+      // force redeploy of AddressSortedLinkedListWithMedian for CR11
+      // since it was deployed by Mento team with different settings and bytecode
+      backward.report.libraries.AddressSortedLinkedListWithMedian = {} as CategorizedChanges
```

Comment:
Without this CR11 will not correctly verify previous release & generate new prop (since bytecodes do not match).  It is also dependent on CR11 to checkout Mento repo. This is rather specific to only CR11 - I would prefer not to propagate it to any public facing flags.

Reply:
On second though, we might want to add this as a new feature, but it should be in different task since this is currently blocking CR11 release